src/Microsoft.AspNet.Mvc.TagHelpers/ImageTagHelper.cs

```diff
@@ -61,9 +63,13 @@ public ImageTagHelper(IHostingEnvironment hostingEnvironment, IMemoryCache cache
 
         protected IMemoryCache Cache { get; }
 
+        protected IUrlHelper UrlHelper { get; }
+
         /// <inheritdoc />
         public override void Process(TagHelperContext context, TagHelperOutput output)
         {
+            Src = UrlHelper.Content(Src);
```

Comment:
Do we want to be smarter about these values? Aka handle whitespace much like the `UrlResolutionTagHelper` does? Question spurred from convo: https://github.com/aspnet/Mvc/pull/2875/files#r35715590

I'm torn between doing the extra computation and keeping the users value as they provided it (since executing a `TagHelper` is more like executing a method).

/cc @DamianEdwards